Tools/MonoGame.Tools.Tests/MonoGame.Tools.Tests.csproj

```
@@ -3,13 +3,15 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>This package contains all the Unit Tests for the MonoGame Framework.</Description>
```

```suggestion
    <Description>The unit tests for the MonoGame framework.</Description>
```

Don't really need a description for this one, but should be consistent if we add it.

---

It is more for consistency